### PR TITLE
R4R cleanup error reporting

### LINKED ISSUE #242: remove codespace.go and replace with res.Log
`codespace.go` currently has all the ibc errors and their messages. This will be very hard to maintain. 

The funcs that use `GetCodespace` have a `sdk.TxReponse` which has both [parsed](https://github.com/cosmos/cosmos-sdk/blob/6d7fc911c28a8929ec09818f4d10a4b8295de3b6/types/result.go#L92) and raw logs. These logs when run in the SDK are created from calling [ABCIInfo](https://github.com/cosmos/cosmos-sdk/blob/6d7fc911c28a8929ec09818f4d10a4b8295de3b6/types/errors/abci.go#L61) and creating an [abci response](https://github.com/tendermint/tendermint/blob/c0682a3bed837d2e7e1f0def91ae25f4dbf34cac/rpc/core/types/responses.go#L182) which is stored in the tm ResultTx which is passed into sdk [TxResponse](https://github.com/cosmos/cosmos-sdk/blob/6d7fc911c28a8929ec09818f4d10a4b8295de3b6/types/result.go#L87).

I am pretty sure we can remove `codespace.go` and simply replace it with the `res.RawLog` or perhaps the parsed logs for neat error reporting (though I am not certain how this error response would look)

### PR DESCRIPTION
**Pull request by colin-axner**
closes #242 

without debug:
```
$ rly tx transfer ibc0 ibc0 10000n0token true $(rly keys show ibc1 testkey) 
I[2020-05-14|12:08:27.262] ✘ [ibc0]@{43483} - msg(0:transfer) err(sdk:5:failed to execute message; message index: 0: 0transfer/ibconexfer/n0token is smaller than 10000transfer/ibconexfer/n0token: insufficient funds) 
Error: failed to send first transaction
```

with debug:
```
$ rly tx transfer ibc0 ibc0 10000n0token true $(rly keys show ibc1 testkey) --debug
I[2020-05-14|12:08:24.024] - [ibc0] -> sending transaction:             
[{"type":"ibc/transfer/MsgTransfer","value":{"source_port":"transfer","source_channel":"ibconexfer","destination_height":"43478","amount":[{"denom":"transfer/ibconexfer/n0token","amount":"10000"}],"sender":"cosmos1hr348hu9s824ukp7h4v4dcuy28jwy92aa60q8n","receiver":"cosmos1dl9sftjsujla8hcfqj7ppj4hfdmhqx3jjpjvh9"}}]
I[2020-05-14|12:08:24.025] ✘ [ibc0]@{43480} - msg(0:transfer) err(sdk:5:failed to execute message; message index: 0: 0transfer/ibconexfer/n0token is smaller than 10000transfer/ibconexfer/n0token: insufficient funds) 
I[2020-05-14|12:08:24.025] - transaction response:                      
{"height":"43480","txhash":"65B8DA71E05702D28BB6D686FE2201AD5375E36A4306914C732D40EA47981193","codespace":"sdk","code":5,"raw_log":"failed to execute message; message index: 0: 0transfer/ibconexfer/n0token is smaller than 10000transfer/ibconexfer/n0token: insufficient funds","gas_wanted":"200000","gas_used":"51034"}
Error: failed to send first transaction
```

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -113,7 +112,6 @@ func Execute() {
 	}
 
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Println(err)
```
Comment (colin-axner) [inline cmd/root.go:116]: this was double printing the error like so:
```
Error: failed to send first transaction
failed to send first transaction
```

Now it just does the first line

**Comment by colin-axner** [comment]
this updates all regularly logged errors to report in full. While not as clean on the log, it is much more useful in identifying quickly what caused the error. The debug flag releases a ton of information and is not likely to be used in common cases. The current error reporting doesn't provide any contextual information on how the error occurred.  Also, what is the workflow for integration testing?

cc: @jackzampolin

File: cmd/root.go (hunk 1 above)
Comment (jackzampolin) [reply cmd/root.go:116]: Thank you for finding this!

**Review by jackzampolin — APPROVED** [review]
This looks good! Thank you for the updates @colin-axner

**Comment by jackzampolin** [comment]
@colin-axner See details on the integration testing framework here: https://github.com/iqlusioninc/relayer/blob/master/docs/testing.md Happy to jump on a call an run throught it.

**Pull request merged**